### DISCUSSION TITLE
How to compress baota-v1.tar for more than 5G with docker-slim command?

### DISCUSSION THREAD

**Discussion by JHagoo** [slimtoolkit/slim#451] [Q&A]
![6B02C304-6D67-4C21-86A7-CAFE33C8C74A](https://user-images.githubusercontent.com/121721737/211342939-fd25a157-2fc5-4ad8-9474-6ad0bfd9ccfc.jpeg)

**Comment by kcq** [comment]
What is `baota-v1.tar`? It doesn't appear to be a regular container image. Is it an image you saved to a tar file from Docker?
